IPython/core/tests/test_magic.py

```diff
@@ -611,8 +611,8 @@ def test_extension():
     try:
         _ip.ipython_dir = tmpdir.name
         nt.assert_raises(ImportError, _ip.magic, "load_ext daft_extension")
-        url = os.path.join(os.path.dirname(__file__), "daft_extension.py")
-        _ip.magic("install_ext %s" % url)
+        url = os.path.join(os.path.dirname(__file__), "daft_extension")
+        _ip.system("pip install %s" % url)
```

Comment:
good point. I home that this syntax will not [go away](https://mail.python.org/pipermail/distutils-sig/2015-November/027536.html)